java/feathub-udf/flink-udf/src/main/java/com/alibaba/feathub/flink/udf/aggregation/AggFuncUtils.java

```diff
@@ -20,19 +20,19 @@
 
 /** Utility of aggregation functions. */
 public class AggFuncUtils {
-    public static AggFunc<?, ?> getAggFunc(String aggFunc, DataType inDataType) {
+    public static AggFunc<?, ?, ?> getAggFunc(String aggFunc, DataType inDataType) {
```

Comment:
Instead of making an assumption in this method, I think it may be better to change the name of those aggregation functions and let the Feathub python SDK choose the correct aggregation function accordingly. The logic is added at `sliding_window_utils._get_agg_func_name_after_pre_agg`. What do you think?